cmd/livepeer/livepeer.go

```diff
@@ -183,19 +191,23 @@ func main() {
 	} else if transcoder {
 		n.NodeType = core.TranscoderNode
 		glog.Info("***Livepeer is in transcoder mode ***")
-		server.RunTranscoder(n, *orchAddr)
+		server.RunTranscoder(n, orchAddresses[0])
 		return
 	} else {
 		n.NodeType = core.BroadcasterNode
 	}
 
 	if n.NodeType == core.BroadcasterNode {
-		if *orchAddr == "" {
+		if orchAddresses[0] == "" {
 			glog.Info("No orchestrator specified; transcoding will not happen")
```

Comment:
This is probably where you want to create `NewOffchainOrchestratorFromOnchainList`